jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -129,7 +129,7 @@ function configure_log_elasticsearch() {
 casc_yml="${_JENKINS_CASC_D}/casc.yml"
 build_xml="${_JENKINS_HOME}/jobs/${JOB_NAME:-job}/builds/${RUN_NUMBER:-1}/build.xml"
 
-truncate --no-create --size 0 /dev/termination-log || exit 1
+truncate -c -s 0 /run/termination-log || exit 1
```

Comment:
Why using the short options? This way it's less understandable.

Reply:
Because this uses the busybox truncate which has not long options. I don't think it makes sense to install gnu truncate just to get long options.

Reply:
ok.